package.json

```diff
@@ -17,7 +17,7 @@
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "start:local-api": "ember serve --environment=local-api",
+    "start:local-api": "ember serve --environment=devlocal",
```

Comment:
@TylerMatteo I left this change in from when you helped me set up the local environment, thought it'd be helpful to update but if we should change it back I'm happy to do so

Reply:
Go ahead and change it back. It's not ideal but `local-api` is helpful for when you _do_ want to use local instances of all APIs.